recipes/newrelic/infrastructure/network/ktranslate-netflow.yml

```diff
@@ -0,0 +1,280 @@
+            echo "FLOW_TYPE env var must be set" >&2
+            exit 47
+          fi
+
+    setup_packagecloud_repository:
+      cmds:
+        - |
+          if [ "{{.PLATFORM_FAMILY}}" == "debian" ]; then
+            curl -s https://packagecloud.io/install/repositories/kentik/ktranslate/script.deb.sh | sudo bash > /dev/null
+          else
+            curl -s https://packagecloud.io/install/repositories/kentik/ktranslate/script.rpm.sh | sudo bash
+          fi
+      silent: true
+
+    install_ktranslate:
+      cmds:
+        - |
+          if [ "{{.PLATFORM_FAMILY}}" == "debian" ]; then
+            sudo apt-get install ktranslate -y > /dev/null
+          else
+            sudo yum install ktranslate -q -y > /dev/null
+          fi
+      slient: true
+
+    setup_env:
+      cmds:
+        - |
+          # Remove existing environment config
+          if [ -f /etc/default/ktranslate.env ]; then
+            sed -i "/^NR_ACCOUNT_ID/d" /etc/default/ktranslate.env
+            sed -i "/^NEW_RELIC_API_KEY/d" /etc/default/ktranslate.env
+            sed -i "/^NR_REGION/d" /etc/default/ktranslate.env
+            sed -i "/^KT_FLAGS/d" /etc/default/ktranslate.env
+          else
+            touch /etc/default/ktranslate.env
+          fi
+        - |
+          # Add our default config with populated variables
+          if [ $(echo {{.NEW_RELIC_REGION}} | grep -i staging | wc -l) -gt 0 ]; then
+            echo 'NR_REGION="us_stage"' >> /etc/default/ktranslate.env
+          # todo: check for EU/GOV regions
+          else
+            echo 'NR_REGION="us"' >> /etc/default/ktranslate.env
+          fi
+
+          echo 'NR_ACCOUNT_ID="{{.NEW_RELIC_ACCOUNT_ID}}"' >> /etc/default/ktranslate.env
+          echo 'NEW_RELIC_API_KEY="{{.NEW_RELIC_LICENSE_KEY}}"' >> /etc/default/ktranslate.env
+          echo 'KT_FLAGS="-snmp /etc/ktranslate/snmp-base.yaml -listen=127.0.0.1:8081 -metrics=jchf -flow_only=true -nf.source={{.FLOW_TYPE}} -tee_logs=true -service_name={{.FLOW_TYPE}} nr1.flow"' >> /etc/default/ktranslate.env
+
+          chown ktranslate:ktranslate /etc/default/ktranslate.env
+    configure_snmp_base:
+      cmds:
+        - |
+          # TODO: determine if we need to take a CLI arg and move snmp-base from a tmp file to /etc/ktranslate/snmp-base.yaml
+
+          # ktranslate must be owner so the ktranslate service can write new configs
+          chown ktranslate:ktranslate /etc/ktranslate/snmp-base.yaml
+
+    restart:
+      cmds:
+        - |
+          if [ {{.IS_SYSTEMCTL}} -gt 0 ]; then
+            systemctl restart ktranslate
+          else
+            if [ {{.IS_INITCTL}} -gt 0 ]; then
+              initctl restart ktranslate
+            # else
+            #   /etc/init.d/ktranslate restart
+            fi
+          fi
+      vars:
+        IS_SYSTEMCTL:
+          sh: command -v systemctl | wc -l
+        IS_INITCTL:
+          sh: command -v initctl | wc -l
+
+    assert_agent_started:
```

Comment:
FWIW we recently removed this similar status check block from the infra recipes and are relying on assert_agent_status_ok instead.  `sleep 10` seemed to be a bit short and its removal fixed a handful of timeout errors

Reply:
ahhh, yeah, thanks for the call out. I noticed that late last night and forgot to make a note to remove that from these recipes